your_first_robot/part1/post-es.md

```diff
@@ -0,0 +1,82 @@
+  * ¿Cómo se arranca el sistema sin tener que entrar por SSH? ¿Cómo compartir su producto final con sus amistades?
+
+Como mencioné, el las hojas de trabajo de CamJam nos llevarán lejos hacia lo que necesitamos aprender para esta serie. Sin embargo, en lugar de usar Raspbian, vamos a usar Ubuntu Core. Las hojas de trabajo aún son perfectamente aplicables, y lo probaré guiándole a través de la configuración de Ubuntu Core en su Raspberry Pi, y luego seguiremos una de las hojas de trabajo de CamJam. Note que esta también es una serie de vídeos, siéntase libre de ver la versión en vídeo de este post (con subtítulos en Español):
+
+[![Vídeo: Su primer robot, parte 1: una guía introductoria a ROS y Ubuntu Core](http://img.youtube.com/vi/KidVVqbsIHI/0.jpg)](http://www.youtube.com/watch?v=KidVVqbsIHI)
+
+## Prerrequisitos
+
+Aunque esto es una introducción, quiero ser claro sobre algunas de las cosas que estoy asumiendo. Esta serie asume que usted está familiarizada un poco con Ubuntu (o alguna otra distro basada en Debian, como Raspbian), en particular, usando la línea de comandos. No espero que sea una profesional escriiendo scripts, pero debería saber cómo navegar el sistema de archivos y usar editores de terminal (vi, nano, etc.)
+
+## ¿Qué es Uuntu Core?
+
+Ubuntu Core es una distribución especializada de Ubuntu que está dedicada a dispositivos que entran en saco de «Internet de las cosas» (IoT, por sus siglas en inglés). Esto incluye enrutadores, termostatos y, por supuesto, robots. Espero que, al introducirle al sistema operativo (y herramientas como ROS) que utilizan las profesionales para hacer su robots, ¡el camino quede listo para que usted logre grandes cosas en este campo!
+
+Ubuntu Core es un poco diferente a Raspbian. ¡Es incluso un poco diferente a Ubuntu clásico (o Ubuntu MATE)! Empecemos instalándolo, y explicaré a qué me refiero.
+
+## Step 1: Instalar Ubuntu Core
+
+Esto ya está bien documentado, pero quiero agregar unas cuantas notas antes de que siga los pasos. Antes que todo, se requiren un teclado y un monitor, pero solo para el primer arranque. A diferencia de Raspbian o Ubuntu MATE, no hay nombre de usuario ni contraseña predefinidas, o un proceso de instalación en el que se cree un usuario. Ni siquiera utiliza un ratón-- esto es más como Ubuntu Server. De hecho, no hay un inicio de sesión local: todo se hace a través de SSH (una sesión remota a través de la red), con claves criptográficas en lugar de contraseñas. Esto es parte de la filosofía de «seguridad predeterminada» de Ubuntu Core. Hay muchos ejemplos de credenciales predefinidas siendo abusadas (vea la [red de bots Mirai](https://en.wikipedia.org/wiki/Mirai_(malware))), y como un sistema operativo enfocado en IoT, ¡Ubuntu Core no puede tener eso! Esa es la razón por la que verá que una cuenta de Ubuntu SSO y claves SSH son prerrequisitos en la guía de instalación (no se preocupe, ambas son fáciles, ¡y su robot estará super seguro!).
```

Comment:
I would change the word "abusadas", it's the literal meaning of abused. But I think, in spanish there's another word for this, maybe a tech word for this in spanish.

Reply:
@allanesqui The original text uses "abused" as well. I'd incline to leave it as is to match the text 1:1. However I agree that the wording could be improved on both languages to better reflect the meaning of this to a beginner.

Reply:
I'm happy with "abusadas". If you have another option, put it here and we can discuss it.